custom-sign-in/app/LoginScreen.js

```diff
@@ -0,0 +1,146 @@
+﻿/*
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, {Fragment} from 'react';
+import {
+  SafeAreaView,
+  Button,
+  StyleSheet,
+  Text,
+  View,
+  StatusBar,
+  TextInput,
+} from 'react-native';
+import Spinner from 'react-native-loading-spinner-overlay';
+
+export default class LoginScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Login',
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      progress: false,
+    };
+    var OktaAuth = require('@okta/okta-auth-js');
+    var config = {
+      url: 'https://sdk-test.trexcloud.com',
+    };
+
+    this.authClient = new OktaAuth(config);
+    this.userName = '';
+    this.password = '';
```

Comment:
nit: Common pattern is to bind these to `this.state` instead and changing state in the Text Field:

```
<TextInput
  style={styles.textInput}
  placeholder="Password"
  secureTextEntry={true}
  onChangeText={(text) => this.setState({text})}
/>
```